ideascube/library/views.py

```diff
@@ -66,6 +66,13 @@ def get_context_data(self, **kwargs):
 
 class BookDetail(DetailView):
     model = Book
+    def get(self, request, *args, **kwargs):
+        ret = super().get(request, *args, **kwargs)
+        user = request.user
+        if user.is_authenticated() and user.is_staff and not self.object.physical:
```

Comment:
`not self.object.physical` will add the warning even for a book that has only digital specimens, which is not what we want AFAIK.